domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -2219,52 +2222,435 @@ public void selectDistinctOn(TestContext context) throws IOException {
   }
 
   @Test
-  public void streamGet(TestContext context) throws IOException {
+  public void streamGetLegacy(TestContext context) throws IOException {
+    AtomicInteger objectCount = new AtomicInteger();
+    final String tableDefiniton = "id UUID PRIMARY KEY , jsonb JSONB NOT NULL, distinct_test_field TEXT";
+    createTableWithPoLines(context, MOCK_POLINES_TABLE, tableDefiniton);
+    postgresClient.streamGet(MOCK_POLINES_TABLE, new Object(), "jsonb", null, false, null,
+      streamHandler -> objectCount.incrementAndGet(), context.asyncAssertSuccess(asyncResult ->
+        context.assertEquals(6, objectCount.get())));
+  }
+
+  @Test
+  public void streamGetLegacyFilter(TestContext context) throws IOException, FieldException {
+    AtomicInteger objectCount = new AtomicInteger();
+    final String tableDefiniton = "id UUID PRIMARY KEY , jsonb JSONB NOT NULL, distinct_test_field TEXT";
+    createTableWithPoLines(context, MOCK_POLINES_TABLE, tableDefiniton);
+    CQLWrapper wrapper = new CQLWrapper(new CQL2PgJSON("jsonb"), "edition=First edition");
+    postgresClient.streamGet(MOCK_POLINES_TABLE, new Object(), "jsonb", wrapper, false, null,
+      streamHandler -> objectCount.incrementAndGet(), context.asyncAssertSuccess(asyncResult ->
+        context.assertEquals(3, objectCount.get())));
+  }
+
+  @Test
+  public void streamGetLegacySyntaxError(TestContext context) throws IOException, FieldException {
+    final String tableDefiniton = "id UUID PRIMARY KEY , jsonb JSONB NOT NULL, distinct_test_field TEXT";
+    createTableWithPoLines(context, MOCK_POLINES_TABLE, tableDefiniton);
+    CQLWrapper wrapper = new CQLWrapper(new CQL2PgJSON("jsonb"), "edition=");
+    AtomicInteger objectCount = new AtomicInteger();
+    postgresClient.streamGet(MOCK_POLINES_TABLE, new Object(), "jsonb", wrapper,
+      false, null, streamHandler -> objectCount.incrementAndGet(), context.asyncAssertFailure());
```

Comment:
Can you replace `objectCount.incrementAndGet()` by `context.fail()` and remove `objectCount`?